src/mame/sega/m1comm.cpp

```diff
@@ -315,6 +316,9 @@ void m1comm_device::comm_tick()
 {
 	if (m_linkenable == 0x01)
 	{
+		std::error_condition filerr;
+		uint64_t filesize; // unused
```

Comment:
Is there a good reason for lifting the unused file size here?  It makes the compiler’s job harder, because it needs to ensure the address is fixed every time a reference to it is passed to a function, and that the value isn’t nuked between potential uses.  Keeping them in the smallest scope possible lets the compiler generate more efficient code.

Reply:
should be fixed now.